nodes/mainnet.md

```diff
@@ -126,6 +126,7 @@ to participate in Mainnet Beta:
 - `rpc-celestia.staker.space`
 - `celestia-rpc.noders.services`
 - `celestia.moonli.me`
+- `celestia-mainnet-rpc.itrocket.net`
```

Comment:
what ports are open?

Reply:
443

Reply:
Archive node, discard_abci_responses = false

Reply:
same here: could you add this in bullets underneath please?